source/html/renderer/src/card-elements.ts

```diff
@@ -3253,12 +3299,16 @@ export class AdaptiveCard extends ContainerWithActions {
         if (!this.isVersionSupported()) {
             renderedCard = document.createElement("div");
             renderedCard.innerHTML = this.fallbackText ? this.fallbackText : "The specified card version is not supported.";
-
-            return renderedCard;
         }
         else {
-            return super.render();
+            renderedCard = super.render();
+            renderedCard.tabIndex = 0;
+            if (!Utils.isNullOrEmpty(this.speak)) {
```

Comment:
Empty line before if

Reply:
No, text block doesn't require aria label. It can be picked up by narrators.